source/index.html

```diff
@@ -1,12 +1,207 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Мишка</title>
   </head>
-  <body>
+  <body class="page__body">
+    <header class="page-header">
+      <a class="page-header__logo" href="#">
+        <img class="page-header__logo-image" src="img/mishka-logo-mobile.svg" width="86" height="35" alt="Мишка.">
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__button" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+          <img class="main-nav__icon" src="img/menu.svg" width="20" height="14" alt="Открыть меню.">
+        </button>
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item">
+            <a class="site-list__link" href="catalog.html">Каталог товаров</a>
+          </li>
+          <li class="site-list__item">
+            <a class="site-list__link" href="form.html">Вязание на заказ</a>
+          </li>
+          <li class="site-list__item site-list__item--bottom">
+            <a class="site-list__link" href="#">Новые поступления</a>
+          </li>
+          <li class="site-list__item site-list__item--bottom">
+            <a class="site-list__link" href="#">Распродажа</a>
+          </li>
+        </ul>
+        <ul class="main-nav__list user-list">
+          <li class="user-list__item">
+            <a class="user-list__link" href="#">Поиск по сайту</a>
+          </li>
+          <li class="user-list__item">
+            <a class="user-list__link" href="#">Корзина: пока пуста</a>
+          </li>
+          <li class="user-list__item user-list__item--bottom">
+            <a class="user-list__link" href="#">Бесплатная доставка по России</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
+    <main class="page-main">
+      <h1 class="visually-hidden">Магазин крафтовых предметов для дома Mishka</h1>
 
+      <section class="promo">
+        <h2 class="promo__title">Милые штуки ручной работы для дома</h2>
+
+        <ul class="promo__list category-list">
+          <li class="category-list__item category-item">
+            <a class="category-item__link" href="#">
+              <p class="category-item__text">Предметы интерьера</p>
+              <img class="category-item__icon" src="img/interior.svg" width="40" height="62" alt="Настольная лампа.">
+            </a>
+          </li>
+          <li class="category-list__item category-item">
+            <a class="category-item__link" href="#">
+              <p class="category-item__text">Детские игрушки</p>
+              <img class="category-item__icon" src="img/toy.svg" width="60" height="60" alt="Кубик Лего.">
+            </a>
+          </li>
```

Comment:
Не очень понятно, есть ли здесь какой-то профит от выделения отдельного блока `category-item`.
Есть ощущение, что и так было бы норм:
```suggestion
          <li class="category-list__item">
            <a class="category-list__link" href="#">
              <p class="category-list__text">Детские игрушки</p>
              <img class="category-list__icon" src="img/toy.svg" width="60" height="60" alt="Кубик Лего.">
            </a>
          </li>
```

А если иконки через стили поставить, то и еще проще будет

Reply:
ок